### PR TITLE
[zh-CN] Added translation of Chapter "matrixsplit"

### PR DESCRIPTION
**Pull request by yarusome**
I found this book really illustrative with a flat learning curve and it's useful for programmers and mathematicians alike. I plan to translate the book into simplified Chinese this year with overhaul for previously translated chapters toward the end since there are many show-stoppers in the existing translation, i.e. "primer" → "底漆" (paint applied at the bottom layer), "implementing" → "实习" (internship).

However, I'm not a programmer by profession and this is my first time using GitHub, so there might be wrong actions done inadvertently.

### CODE REVIEW

**Comment by Pomax** [comment]
More than happy to take translations and corrections - all the translations are community contributions, so if there are typos, or even just bad characters, I'll never know, myself =D

**Comment by yarusome** [comment]
I hope it was not too serious and you could get well soon. I've resumed translation since last week, too. Btw, is there any pending procedure I have to complete to get the workflow approved?

File: docs/chapters/matrixsplit/content.zh-CN.md [hunk 1]
```diff
@@ -0,0 +1,567 @@
...
+  \end{bmatrix}
+  \cdot
+  M
+  \underbrace{ \kern 1.25em \cdot \kern 1.25em Q \kern 1.25em \cdot \kern 1.25em}_{\textit{……得到这一项！}}
+  \begin{bmatrix}
+  P_1 \\ P_2 \\ P_3
+  \end{bmatrix}
+\]
+
+以上变形可行是因为[*M · M<sup>-1</sup>*]为单位矩阵。这有点像在计算中让某项乘以x/x——不改变函数本身，却可以将函数改写为更好处理的形式或者得到不同的分解。类似地将上式中的矩阵左乘以[*M · M<sup>-1</sup>*]不会影响整个式子，却可以将矩阵序列[*something · M*]变为[*M · something*]，而这至关重要——如果知道了[*M<sup>-1</sup> · Z · M*]是什么，那就可以将其施加到已有坐标上得到一条二次贝塞尔曲线的标准矩阵表示（即[*T · M · P*]）)和表示从*t = 0*到*t = z*的曲线的一系列新坐标。计算如下：
```
Comment (Pomax) [inline docs/chapters/matrixsplit/content.zh-CN.md:191]: it probably makes sense to translate "something" to the Chinese equivalent here, too.

**Review by Pomax — CHANGES_REQUESTED** [review]
just one small change request

**Comment by Pomax** [comment]
I finally have a working arm and hand again, so I will doing the outstanding pull request reviews. Apologies that it took so long.

**Pull request closed without merging**